fabric-command-api-v1/src/main/java/net/fabricmc/fabric/api/client/command/v1/FabricClientCommandSource.java

```diff
@@ -55,13 +58,56 @@
 	 * Gets the player that used the command.
 	 *
 	 * @return the player
+	 * @deprecated Please use {@link #getTargetPlayer()} instead.
```

Comment:
```suggestion
	 * @deprecated Please use {@link #getTargetPlayer()} for the user, and {@code getClient().player} for the client's player.
```
Now we actually need to distinguish the two cases.

Reply:
Indeed.